db/schema.rb

```diff
@@ -49,7 +49,7 @@
   add_index "comments", ["journal_id"], name: "index_comments_on_journal_id", using: :btree
   add_index "comments", ["user_id"], name: "index_comments_on_user_id", using: :btree
 
-  create_table "default_addresses", force: true do |t|
+  create_table "default_addresses", force: :cascade do |t|
```

Comment:
[rubocop] Prefer single-quoted strings when you don't need string interpolation or special symbols.